lib/new_relic/manticore/wrapped_request.rb

```diff
@@ -1,3 +1,12 @@
+# Check version to account for a breaking path change
+# introduced in 6.10
+#
+if NewRelic::VERSION::MAJOR == 6 && NewRelic::VERSION::MINOR < 10
```

Comment:
maybe restrict newrelic rpm gem dependency to >= 6 in the gemspec, since with version 5 this would not work

Reply:
ah just saw its already ~> 6 so thats fine

Reply:
on the other hand it probably would be fine to have no backwards compatibility in the code and require in the gemspec "~> 6.10"

Reply:
Yea but then this doesn't really hurt either and is guaranteed to be compatible. I'd rather not  restrict the versions if it's not really necessary